openapi-cli/src/test/java/io/ballerina/openapi/cmd/BallerinaToOASTests.java

```diff
@@ -135,9 +135,7 @@ public void testHttpRequest() {
         try {
             cmd.execute();
             output = readOutput(true);
-            Assert.assertTrue(output.trim().contains("WARNING [http_request.bal:(4:31,4:44)] Generated OpenAPI " +
-                    "definition does not contain request body information of the `GET` method," +
-                    " as its not supported by the OpenAPI specification."));
+            Assert.assertFalse(Files.exists(this.tmpDir.resolve("http_request_openapi.yaml")));
```

Comment:
Any reason why we aren't asserting the error message anymore?

Reply:
Earlier, This error message we set for when `get` method has `@http:Payload` to avoid details generation regarding requestBody. Currently, http module handles this scenario by itself within http module giving a compilation error. Therefore this message was removed.